mumbleBot.py

```diff
@@ -33,13 +34,15 @@
 
 
 class MumbleBot:
-    version = 'git'
```

Comment:
Please keep this line. Azlux has the routine for replacing this line with the correct version when doing a release. The 'git' stands for the development version and will be dealt with https://github.com/azlux/botamusique/blob/014086305d54a2b585839f34e8d279c384658596/mumbleBot.py#L209.